vorc_gazebo/launch/perception_task.launch

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<launch>
+  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find vrx_gazebo)/config/custom_rosconsole.conf"/>
+  <!-- Gazebo world to load -->
+  <arg name="world" default="$(find vorc_gazebo)/worlds/perception_task.world" />
+  <!-- If true, run gazebo GUI -->
+  <arg name="gui" default="true" />
+  <!-- If true, run gazebo in verbose mode -->
+  <arg name="verbose" default="false"/>
+  <!-- Set various other gazebo arguments-->
+  <arg name="extra_gazebo_args" default=""/>
+  <!-- Start in a default namespace -->
+  <arg name="namespace" default="cora"/>
+  <!-- Start paused? -->
+  <arg name="paused" default="false"/>
+
+  <!-- Initial USV location and attitude-->
+  <arg name="x" default="10" />
+  <arg name="y" default="-372" />
+  <arg name="z" default="0.1" />
+  <arg name="P" default="0" />
+  <arg name="R" default="0" />
+  <arg name="Y" default="0.3816" />
+
+  <!-- VORC sensors enabled -->
+  <arg name="vorc_sensors_enabled" default="true" />
```

Comment:
I don't think we're using this parameters as all the sensors are enabled in our case.